project/plugins.sbt

```diff
@@ -1,10 +1,10 @@
-// The Typesafe repository
-resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
-
 // Use the Play sbt plugin for Play projects
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.10")
 
 // http://www.scalastyle.org/sbt.html
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.7.0")
 
-resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
+// sbt revolver
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
```

Comment:
sbt-revolver for automatically restart application if sources have been changed

[sbt-revolver](https://github.com/spray/sbt-revolver)